bundles/AdminBundle/Resources/public/js/pimcore/object/folder/search.js

```diff
@@ -232,10 +232,13 @@ pimcore.object.search = Class.create(pimcore.object.helpers.gridTabAbstract, {
         }
         this.store.getProxy().setExtraParam("only_direct_children", this.onlyDirectChildren);
         this.store.setPageSize(itemsPerPage);
-        if (existingFilters) {
+
+        if (existingFilters && this.classId == this.previousClassId) {
```

Comment:
@weisswurstkanone how about if we keep the exisiting filter in case of `fromConfig=true`? and then everything else should reset the filter. 

Also, we should keep 'Filter active' & 'Clear Filter' buttons on config change.

Reply:
keep it even if the class changes ?

Reply:
I mean:
```
if (!formConfig) {
    this.store.setFilters(existingFilters.items);
}	   
```